js/packages/logger/src/logger.ts

```diff
@@ -36,6 +37,11 @@ export function makeLogger({
   console = true,
   extended = false,
 }: LoggerOptions & {extended?: boolean} = {}): Logger {
+  if (process.env.APPLITOOLS_SHOW_LOGS && process.env.DEBUG) {
+    handler = {type: 'debug', label}
+    level = LogLevel.all
+    format = (chunks: any[], {tags, level}) => defaultFormat(chunks, {level, prelude: true, tags, timestamp: false})
+  }
   if (!handler) {
```

Comment:
no need for this check if the first one was triggered
```suggestion
  else if (!handler) {
```

Reply:
done